lib/grape-swagger/doc_methods.rb

```diff
@@ -491,7 +497,7 @@ def setup(options)
         end
 
         ops = visible_ops.group_by do |route|
-          @@documentation_class.parse_path(route.route_path, api_version)
+          @@documentation_class.parse_path(route.route_path, api_version, hide_module_from_path)
```

Comment:
It would be cool to have them as class vars or class instance vars on each dynamically created documentation class rather than the global module and rather than needing to pass options around all over the place. Ill see if I can come up with a nice approach.